contrib/go-redis/redis.v8/option.go

```diff
@@ -58,3 +59,11 @@ func WithAnalyticsRate(rate float64) ClientOption {
 		}
 	}
 }
+
+// WithExcludeRawCommand sets the option to exclude the raw command
+// for when the command/cache might contain sensitive data
+func WithExcludeRawCommand(exclude bool) ClientOption {
```

Comment:
```suggestion
func SkipRawCommand(skip bool) ClientOption {
```
I think this would be better, unless it's bad that we don't have the `With*` prefix?